### DISCUSSION TITLE
Inlining global CSS?

### DISCUSSION THREAD

**Discussion by KATT** [vercel/next.js#14272] [Help]
Styled JSX is nicely inlined in the page which avoids extra roundtrips etc - is there a way to achieve the same thing with global CSS?

Reference - https://github.com/vercel/next.js/tree/canary/examples/with-next-css

**Comment by revskill10** [comment]
I think in this case we should manually using `raw-loader` for css file, then inline in `<Head>` instead ?

**Reply by KATT** [reply]
I failed to mention it's SCSS so raw-loader wouldn't quite be enough

**Comment by lebreRafael** [comment]
It is possible to do that by overriding `getCssLinks` implementation on `Head`. You can check the implementation [here](https://stackoverflow.com/questions/57057947/how-to-inline-css-in-the-head-tag-of-a-nextjs-project/66430461#66430461). Make sure to read the comment because it has the final/working code
